lifecycle-operator/controllers/common/evaluation/evaluationhandler.go

```diff
@@ -26,9 +27,9 @@ type IEvaluationHandler interface {
 	ReconcileEvaluations(ctx context.Context, phaseCtx context.Context, reconcileObject client.Object, evaluationCreateAttributes CreateEvaluationAttributes) ([]klcv1alpha3.ItemStatus, apicommon.StatusSummary, error)
 }
 
-type EvaluationHandler struct {
```

Comment:
shall we call the file just "handler.go"?

Reply:
good point, yes